tests/test_reader.py

```diff
@@ -281,14 +281,14 @@ def test_hdf5(tmp_path, root):
 
     with HDFStore(filename) as store:
         assert_frame_equal(samples, store[key])
-        assert type(store[key]) == type(samples)
+        assert type(store[key]) is type(samples)
```

Comment:
Possibly `isinstance(store[key], type(samples))` or `isinstance(store[key], Samples)` would be more appropriate. `type(.) is type(.)` is stricter. Not sure which would be better here and in the rest of this test...

Reply:
stricter is better here -- we definitely want the type to not change in between writing and reading.